string EPC
---
int8 result
